### ISSUE TITLE
Correct output format

### ISSUE THREAD

**Issue by slifty** [art-o-matic/walter-tvkitchen-implementation#3]
The walter system ingests caption data that looks like:

```
2021-05-30T22:10:44.000044 OUR ELECTIONS THAT WE NEED TO
2021-05-30T22:01:19.000046 RESTORE.
2021-05-30T22:10:13.000046 >> REPUBLICANS WANT TO MAKE IT
2021-05-30T22:15:47.000047 HARDER TO VOTE AND EASIER TO
2021-05-30T22:09:42.000049 STEAL AN ELECTION.
2021-05-30T22:09:12.000052 >> WE DEFINITELY HAVE TURNED A
2021-05-30T22:05:21.000055 CORNER.
2021-05-30T22:16:29.000055 >> WE THOUGHT WE AROUND GOING
```

This is different from the standard SRT format:

```
00:00:21,950 --> 00:00:22,584
LATEST EPISODE OF MASS GUN

00:00:22,584 --> 00:00:25,888
VIOLENCE IN AMERICA.

00:00:25,888 --> 00:00:28,123
THEN, RACE MATTERS-- THE PASSAGE
```

We'll need to translate SRT payload content to look as expected.

We should also change the file format to be:

`CNN_Newsroom_With_Alisyn_Camerota_and_Victor_Blackwell_caption_21-05-26-19-00-00.txt`

^ note the `caption` in the filename.

**Comment by slifty** [comment]
New output format:

```
2021-05-31T16:36:52.740 SHE LOVES TO GET UNDER
2021-05-31T16:37:12.527 >> John: THERE'S A FINE LIN FOR
2021-05-31T16:37:18.466 SOMEONE LIKE OSTAPENKO WHO LOVES
2021-05-31T16:37:20.234 TO GO AFTER THE BALL, BUT IT'S
2021-05-31T16:37:23.004 GOT TO  FRUSTRATING FOR HER TO
2021-05-31T16:37:24.939 MAKE SO MANY ERRORS.
```

**Comment by slifty** [comment]
Closed by #5
